web/src/components/customer/explore/index.tsx

```diff
@@ -46,6 +47,15 @@ const CustomerExplorePage: React.FC = () => {
     setNumCommits(numOngoingCommits);
   };
 
+  const handleGetIdentity = async () => {
+    const {
+      idToken,
+      decodedToken: {sub},
```

Comment:
What does this "sub" mean here?

Reply:
@chshda `sub` is the unique identifier for gpay users! Can refer to https://developers.google.com/pay/spot/eap/reference/identity-api#identitytoken the `sub` field